prettyPrinters/providers.py

```diff
@@ -457,10 +457,16 @@ class StdRefSyntheticProvider:
     def __init__(self, valobj, dict, is_cell=False):
         # type: (SBValue, dict, bool) -> StdRefSyntheticProvider
         self.valobj = valobj
-        self.borrow = valobj.GetChildMemberWithName("borrow").GetChildAtIndex(0).GetChildAtIndex(0)
-        if not is_cell:
-            self.borrow = self.borrow.GetChildAtIndex(0)
-        self.value = valobj.GetChildMemberWithName("value").GetChildAtIndex(0)
+
+        if is_cell:
+            self.borrow = valobj.GetChildMemberWithName("borrow").GetChildMemberWithName(
+                "value").GetChildMemberWithName("value")
+            self.value = valobj.GetChildMemberWithName("value").GetChildMemberWithName("value")
+        else:
+            self.borrow = valobj.GetChildMemberWithName("borrow").GetChildMemberWithName(
+                "borrow").GetChildMemberWithName("value").GetChildMemberWithName("value")
+            self.value = valobj.GetChildMemberWithName("value").GetChildWithName("value").GetChildWithName(
+                "value").Dereference()
```

Comment:
Looks like it doesn't work, at least for me
<img width="526" alt="screen shot 2018-12-01 at 02 45 15" src="https://user-images.githubusercontent.com/2539310/49320546-3a886080-f513-11e8-8553-515cba5e5949.png">

Reply:
Oh, I'm sorry, I've forgotten to push the last changes. Fixed.

<img width="527" alt="screenshot 2018-12-01 at 20 39 51" src="https://user-images.githubusercontent.com/4854600/49331105-46befd00-f5a9-11e8-98c4-d111c4d8973a.png">